autoreduce_frontend/templates/snippets/variables/render.html

```diff
@@ -6,24 +6,32 @@
     <label for="var-{{ type }}-{% encode_b64 name %}" class="col-form-label col-12">
     {{ name }}
     {% if name in variable_help %}
-    <a href="#" data-toggle="popover" data-html="true" data-content="{% dict_get variable_help name %}"
-        data-trigger="hover" data-placement="top" data-container="body">
-        <i class="fa fa-info-circle"></i>
-    </a>
+        <a href="#" data-toggle="popover" data-html="true" data-content="{% dict_get variable_help name %}"
+            data-trigger="hover" data-placement="top" data-container="body">
+            <i class="fa fa-info-circle"></i>
+        </a>
     {% endif %}
     </label>
     <div class="form-row col-sm-12">
         <div class="col-11">
             {% if variable.current|lower == "true" or variable.current|lower == "false" %}
-            <select id="var-{{ type }}-{% encode_b64 name %}" data-initial="{{ variable.current }}" data-default="{{ variable.default }}" name="var-{{ type }}-{% encode_b64 name %}">
-            <option value="True" {% if value.lower == 'true' %} selected {% endif %}>True</option>
-            <option value="False" {% if value.lower == 'false' %} selected {% endif %}>False</option>
-            </select>
+                <select id="var-{{ type }}-{% encode_b64 name %}" data-initial="{{ variable.current }}" data-default="{{ variable.default }}" name="var-{{ type }}-{% encode_b64 name %}">
+                    <option value="True" {% if value.lower == 'true' %} selected {% endif %}>True</option>
+                    <option value="False" {% if value.lower == 'false' %} selected {% endif %}>False</option>
+                </select>
+            {% elif "file" in name %}
```

Comment:
This will only work for variable with "file" in their name, it should be something like `if "all_files" in variable`